src/gc.c

```diff
@@ -234,6 +236,7 @@ void jl_gc_wait_for_the_world(jl_ptls_t* gc_all_tls_states, int gc_n_threads)
     TracyCZoneColor(ctx, 0x696969);
 #endif
     assert(gc_n_threads);
+    safepoint_waiter = uv_thread_self();
```

Comment:
This needs to be unset also?

Reply:
Done! See `memset` at the end of `jl_gc_wait_for_the_world`.